src/neo/Network/P2P/Payloads/Transaction.cs

```diff
@@ -464,6 +464,7 @@ public StackItem ToStackItem(ReferenceCounter referenceCounter)
                 NetworkFee,
                 ValidUntilBlock,
                 Script,
+                new Array(referenceCounter, Signers.Select(u => new ByteString(u.ToArray())))
```

Comment:
Can you change it to a Map[hash=>scope] ?

Reply:
Done